src/graspit_commander/graspit_commander.py

```diff
@@ -51,10 +59,12 @@
     InvalidBodyPoseException,
     BodyCollisionException,
     ClearWorldException,
-    LoadWorldException
+    LoadWorldException,
+    ImportException,
+    SaveImageException,
+    SaveWorldException,
 )
 
-
```

Comment:
try to keep consistent with PEP8 formatting (2 new lines between the imports and your functions)